packages/api/src/joinCommunity.ts

```
@@ -1,15 +1,15 @@
 import { Ad4mClient } from '@coasys/ad4m';
 
 export interface Payload {
   joiningLink: string;
+  client: Ad4mClient;
 }
 
-export default async ({ joiningLink }: Payload): Promise<Community> => {
+export default async ({ joiningLink, client }: Payload): Promise<Community> => {
   try {
-    const client: Ad4mClient = await getAd4mClient();
+    const client: Ad4mClient = client;
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)